### PR TITLE
[MediaVision] Fix an issue that callback was made on a garbage collected delegate

### PR DESCRIPTION
**Pull request by tyoungroy**
Signed-off-by: Tae-Young Chung <ty83.chung@samsung.com>

### Bugs Fixed ###
- TC is sometimes block because of issue that callback was made on a garbage collected delegate.
- To fix it, declare the member variable for callback.

### CODE REVIEW

File: src/Tizen.Multimedia.Vision/MediaVision/SurveillanceEngine.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ public abstract class SurveillanceEngine : IDisposable
         private IntPtr _handle = IntPtr.Zero;
         private bool _disposed = false;
 
+        internal EventOccurredCallback OnEventDetected;
```
Comment (hsgwon) [inline src/Tizen.Multimedia.Vision/MediaVision/SurveillanceEngine.cs]: How about use the word ```_eventDetectedCallback```? It seems like eventHandler to me.

Comment (hsgwon) [inline src/Tizen.Multimedia.Vision/MediaVision/SurveillanceEngine.cs]: It seems that Recognized and Detected events in several classes are using this sindle field together. Is it OK?

Comment (hsgwon) [reply src/Tizen.Multimedia.Vision/MediaVision/SurveillanceEngine.cs]: I missed that all those classes are not just seperate instance but base and derived class relation.  It's OK.

**Comment by tyoungroy** [comment]
I change "OnEventDetected" to "_eventDetectedCallback" and update commit.

**Pull request merged**